templates/components/autocompleteinteractionhandler.js.sapphire

```diff
@@ -0,0 +1,39 @@
+{
+	"category": "interaction-handlers",
+}
+---
+const { InteractionHandler, InteractionHandlerTypes } = require('@sapphire/framework');
```

Comment:
For all template files use the Sapphire prettier config to format them. Copy the template code to [the prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6VACAwhAJnDACzgCc4QAaECABxgEtoBnZUAQxJIgHcAFDhCxRsANtzYBPFlQBGJNmADWcGAGU2AWzgAZelHJIAZqKZxZ8pStU0FegObIYJAK5mQcDTLi58ubWyg7ZzY7OAAxCBINNhgGQOQQNmcYCEoQQhgNEQB1Qnp4JhswOFVBfPoAN3yJBLAmaRA9UxIYXnk7aORjEVMqACsmAA9VexE4AEVnCHgukzcbEmaEmAkaOCYwEno6NJot2Gz6XBhCZABGAFYABio9iFNs+RoEvfXSCvIqAEcp+DbaISJJgAWn03m8aTIP3oZDaIU6RjmVFMGnojhcbiYowmvwMTlcVBgbBkh2OpyQABZCfJ6CJ7DgNAiQFBoJ8QM5TAAVYlCfFuCquACSUHwsFUm22MAAgiLVCsxrMenAAL7KoA) then paste the result back.

Also for all JS template files please add JSDoc comments similar to the already existing ones to add some basic types to JS code

Reply:
So adding
```
	/**
		* @param {InteractionHandler.Context} context
	*/
``` to every JS interaction-handler template

Reply:
Yes